src/main/java/com/ericsson/ei/handlers/RmqHandler.java

```diff
@@ -171,16 +198,59 @@ Binding binding() {
     }
 
     @Bean
-    public List<Binding> bindings() {
+    public List<Binding> bindings(){
         final String[] bingingKeysArray = splitBindingKeys(bindingKeys);
         final List<Binding> bindingList = new ArrayList<>();
         for (final String bindingKey : bingingKeysArray) {
             bindingList.add(BindingBuilder.bind(externalQueue()).to(exchange()).with(bindingKey));
         }
+        deleteBindings(bingingKeysArray,bindingList);
         return bindingList;
     }
 
-    @Bean
+    private void deleteBindings(String[] bingingKeysArray, List<Binding> bindingList){
```

Comment:
Can you put better variable names on input parameter "String[] bingingKeysArray, List<Binding> bindingList" , so its easier to understand which bindings is new bindings and what difference between those.
Or write Java doc.
Example of names , e.g:
"newBindingKeysArray"
"AMQPBindingObjectList"

Or something similar.